01-JS-DOM-APIs/app.js

```diff
@@ -0,0 +1,81 @@
+//--------------excercise 1--------------------//
+
+let section = document.getElementById('hidden');
+window.onload = () => section.style.opacity = 1;
+
+//------------excercise 3 (2 before)----------//
+
+let getJoke = async() => {
+    let jokeSection = document.getElementById('jokeSection');
+    try {
+        let response = await fetch('http://api.icndb.com/jokes/random');
+        let joke = await response.json();
+        chuckJoke.innerHTML = joke.value.joke;
```

Comment:
rocordar modificar esta variable global por un getElementById

```suggestion
        const chuckJoke = document.getElementById('chuckJoke');
        chuckJoke.innerHTML = joke.value.joke;
```